node/node_handler.go

```diff
@@ -66,25 +66,7 @@ func (node *Node) HandleNodeMessage(
 	case proto_node.Block:
 		switch blockMsgType := proto_node.BlockMessageType(msgPayload[0]); blockMsgType {
 		case proto_node.Sync:
-			blocks := []*types.Block{}
```

Comment:
We may still need some logic in this part because instead of broadcasting every block from beacon chain, now only the epoch blocks need to be broadcasted and processed here.